### ISSUE TITLE
Implement `query_list` function

### ISSUE THREAD

**Issue by delgrecoj** [Accelegator/accelegator#29]
(scaffolded in the `queries.py` module, waiting on merge of pull request #27 which addresses issue #26)

**Comment by delgrecoj** [comment]
@mendezjw, @TravisThomasAC -- status update on this issue?

**Comment by TravisThomasAC** [comment]
Will update after work is done tonight.

**Comment by delgrecoj** [comment]
Just for clarification, this function needs to take in the dataframe and return a list, containing the emails of every response in the dataframe.
Updated return signatures can be found in the branch for issue #31.

**Comment by TravisThomasAC** [comment]
Found a stack overflow link that may be helpful for finding values in a data frame
https://stackoverflow.com/questions/42386629/pandas-find-index-of-value-anywhere-in-dataframe

**Comment by TravisThomasAC** [comment]
Also, the link to pandas.index API,
https://pandas.pydata.org/pandas-docs/stable/generated/pandas.Index.html

**Comment by TravisThomasAC** [comment]
Pushed an implementation of `query_list` function

**Comment by delgrecoj** [comment]
merged into branch for #31 and passed testsuite; closing.
